imapclient/client.go

```diff
@@ -45,11 +45,9 @@ const (
 
 	cmdWriteTimeout     = 30 * time.Second
 	literalWriteTimeout = 5 * time.Minute
-)
 
-var dialer = &net.Dialer{
-	Timeout: 30 * time.Second,
-}
+	defaultDialTimeout = 30 * time.Second
```

Comment:
I would prefer to make `defaultDialTimeout` public, wdyt?

Reply:
Why?

Reply:
- allows to use the upstream defaultDialTimeout and keeping it in-sync when passing a custom net.Dialer to only change another option than the timeout,
- allows to base other timeouts on it while using the default without having to pass custom Options,
- allows to log the timeout without having to set it explicitly to ensure it stays in-sync